workflow/applications/gfs_cycled.py

```diff
@@ -230,12 +231,16 @@ def get_task_names(self):
 
         if self.do_gempak:
             gfs_tasks += ['gempak']
+            gfs_tasks += ['gempakmeta']
+            gfs_tasks += ['gempakncdcupapgif']
+            gfs_tasks += ['gempakpgrb2spec']
+            gfs_tasks += ['npoess_pgrb2_0p5deg']
 
         if self.do_awips:
             gfs_tasks += ['awips_20km_1p0deg', 'awips_g2', 'fbwinds']
 
-        if self.do_npoess:
-            gfs_tasks += ['npoess']
+        if self.do_wafs:
+            gfs_tasks += ['wafs', 'wafsgcip', 'wafsgrib2', 'wafsgrib20p25', 'wafsblending', 'wafsblending0p25']
```

Comment:
WAFS was removed, and npoess shouldn't have been.
```suggestion
        if self.do_npoess:
            gfs_tasks += ['npoess_pgrb2_0p5deg']
```

Reply:
See https://github.com/NOAA-EMC/global-workflow/pull/2103#discussion_r1415841460